helm-charts/iqserver/values.yaml

```diff
@@ -5,7 +5,7 @@
 
 iq:
   name: nxiq
-  imageName: registry.connect.redhat.com/sonatype/nexus-iq-server:1.99.0-ubi-1
+  imageName: registry.connect.redhat.com/sonatype/nexus-iq-server:1.100.0-ubi-2
```

Comment:
@jflinchbaugh just curious why there's a '2' here instead of a '1'?

Reply:
there had been 2 docker builds kicked off during release of IQ 100. doesn't matter too much. -1 and -2 are likely equivalent, but -2 is latest.